zone/bot.h

```diff
@@ -825,8 +824,7 @@ class Bot : public NPC {
 	eStandingPetOrder m_previous_pet_order;
 	uint32 m_bot_caster_range;
 	BotCastingRoles m_CastingRoles;
-	std::map<std::string,std::string> bot_data_buckets;
-	std::map<std::string,std::string> bot_owner_data_buckets;
+	std::map<std::string,std::string> m_bot_data_buckets;
```

Comment:
It's already on my list to add a general data bucket cache. This already starts to bleed into that.

I would not suggest a `std::map` even though it makes it easier to key, I would suggest a vector and do lookups with a getter because ultimately these caches will be bound to a struct type in the future with a timestamp

Reply:
Would it just be better off if I make it a struct now, like below? or any thoughts on how to pair the values.

```struct DataBucketCache
{
	uint64_t      bucket_id;
	std::string   bucket_key;
	std::string   bucket_value;
	uint32_t      bucket_expires;
};```

Reply:
Pushed up proposed changes to storing the cache.